src/FsCheck/Gen.fs

```diff
@@ -646,12 +646,18 @@ module Gen =
 ///Operators for Gen.
 type Gen<'a> with
 
+    /// Puts a in a Gen.
+    static member Return (a) = Gen.constant a
+    
     /// Lifted function application = apply f to a, all in the Gen applicative functor.
     static member (<*>) (f, a) = apply f a
 
-    /// Like <*>, but puts f in a Gen first.
+    /// Like <*>, but puts f in a Gen first. An alias for Map but with arguments swapped.
     static member (<!>) (f, a) = Gen.constant f <*> a
```

Comment:
I know I wrote this, but now that you've attracted my attention to it :)

I think this should just call `Gen.map` because that's what it is. Also the part of the comment `Like <*>, but puts f in a Gen first.` is confusing (again, my bad). I suggest just keeping your `An alias for map with arguments swapped`.

Reply:
I agree. Done !